File: codeforces/96A/cpp/96a.cpp
#include <iostream>
#include <string>
#include <cstring>
using namespace std;

const char *c_danger_0 = "0000000";
const char *c_danger_1 = "1111111";


int main()
{
    string players;
    cin >> players;

    if (strstr(players.c_str(), c_danger_0) || strstr(players.c_str(), c_danger_1)) {
        cout << "YES" << endl;
    }
    else {
        cout << "NO" << endl;
    }

    return 0;
}

File: codeforces/96A/go/96a.go
package main

import (
    "fmt"
    "strings"
    )

func main() {
    var players string
    fmt.Scan(&players)
    if strings.Contains(players, "0000000") || strings.Contains(players, "1111111") {
        fmt.Println("YES")
    } else {
        fmt.Println("NO")
    }
}
